controller/bin/watch-command

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+tput reset
+
+markdown Changelog.md > Changelog.html
+
+# The controller application requires certain artifacts to be in a specific location relative to binary location. The `dune build @install` command ensures this.
+# The `--profile release` option disables dune from failing on warnings (which are currently present in the `obus` library)
+dune build @install --profile release
+
+if [ "$?" -eq 0 ]; then
+
+  # Kill already running server if any
+  lsof -i tcp:3333 | grep LISTEN | awk '{print $2}' | xargs kill 2>/dev/null
```

Comment:
Is the killing put behind this guard on purpose? It has the effect of letting the previously built server run if the new build fails, which I think could lead to confusion.

Reply:
Exact, fixed with https://github.com/dividat/playos/pull/47/commits/5c00e1e890e31f7cb6810acaebdd5e6ecec48af4. I also improved the initial launch of `bin/dev-server`, after a `dune clean`.